packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -232,13 +232,27 @@ describe('build-schema', () => {
       expect(key).to.equal('modelPartial');
     });
 
-    it('returns concatenated option names otherwise', () => {
+    it('returns "optional[id,_rev]" when a single option "optional" is set', () => {
```

Comment:
The test name is misleading, you are setting two properties as optional.

```suggestion
    it('returns "optional[id,_rev]" when "optional" is set with two items', () => {
```

Reply:
So this is supposed to mean only the `optional` option is set (similar to the `partial` [test](https://github.com/strongloop/loopback-next/blob/e11e73acfd101e34ccc4ca775ef72c3e80d0d3ee/packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts#L230) above it: `returns "partial" when a single option "partial" is set'`), but I'll reword this one.